README.md

```diff
@@ -220,6 +220,41 @@ class MailingAddress < UserData
 end
 ```
 
+#### Returning only the fields you want
+
+After exposing the desired attributes, you can choose which one you need when representing some object or collection, see the example:
+
+```ruby
+class UserEntity
+  expose :id
+  expose :name
+  expose :email
+end
+
+class Entity
+  expose :id
+  expose :title
+  expose :user, using: UserEntity
+end
+
+data = Entity.represent(model, only: 'name,user.name,user.email')
```

Comment:
I really think that `only` should take an array or a single symbol, not a comma-delimited string.